core/src/main/java/me/mykindos/betterpvp/core/client/Rank.java

```diff
@@ -8,38 +8,122 @@
+            null,
+            null,
+            List.of("Standard Player"),
+            0
+    ),
+    YOUTUBE("YT",
+            NamedTextColor.DARK_RED,
+            UtilMessage.deserialize("<white>Y</white><red>T</red>"),
+            UtilMessage.StudioPrefix.appendSpace().append(UtilMessage.deserialize("<white>Y</white><red>T</red>")),
+            List.of("A BetterPvP Content Creator"),
+            1
+    ),
+    HELPER("Helper",
+            NamedTextColor.DARK_GREEN,
+            Component.text("Helper", NamedTextColor.DARK_GREEN),
+            UtilMessage.StudioPrefix.appendSpace().append(Component.text("Helper", NamedTextColor.DARK_GREEN)),
+            List.of("A BetterPvP Helper",
+                    "Ask this person for help!"),
+            2
+    ),
+    TRIAL_MOD("Trial Mod",
+            NamedTextColor.DARK_AQUA,
+            Component.text("T.Mod", NamedTextColor.DARK_AQUA),
+            UtilMessage.StudioPrefix.appendSpace().append(Component.text("T. Mod", NamedTextColor.DARK_AQUA)),
+            List.of("A BetterPvP Trial Moderator",
+                    "This person is trialing to",
+                    "Be a BetterPvP Moderator. You can",
+                    "Ask them for help!"),
+            3
+    ),
+    MODERATOR("Mod",
+            NamedTextColor.AQUA,
+            Component.text("Mod", NamedTextColor.AQUA),
+            UtilMessage.StudioPrefix.appendSpace().append(Component.text("Mod", NamedTextColor.AQUA)),
+            List.of("A BetterPvP Moderator",
+                    "This person is responsible",
+                    "For enforcing BetterPvP chat rules",
+                    "and to help players!"),
+            4
+    ),
+    MINEPLEX("Mineplex",
+            NamedTextColor.GOLD,
+            UtilMessage.MineplexPrefix,
+            UtilMessage.MineplexPrefix,
+            List.of("A Mineplex Staff Member",
+                    "This is a Mineplex Staff member"),
+            5
+    ),
+    ADMIN("Admin",
+            NamedTextColor.RED,
+            Component.text("Admin", NamedTextColor.RED),
+            UtilMessage.StudioPrefix.appendSpace().append(Component.text("Admin", NamedTextColor.RED)),
+            List.of("A BetterPvP Admin",
+                    "This person is a BetterPvP Admin",
+                    "They have more permissions to handle",
+                    "Problems within BetterPvP",
+                    "and typically have more",
+                    "responsibilities within the studio"),
+            6),
+    DEVELOPER("Developer",
+            NamedTextColor.WHITE,
+            Component.text("Dev", NamedTextColor.WHITE),
+            UtilMessage.StudioPrefix.appendSpace().append(Component.text("Dev", NamedTextColor.WHITE)),
+            List.of("A BetterPvP Developer",
+                    "This person is a BetterPvP Developer",
+                    "They create and maintain the games",
+                    "BetterPvP offers on Mineplex"),
+            7);
```

Comment:
Not sure how the weird capitalisation on each new line will look

Reply:
It looks okay, but I have standardized it to work like a sentence https://prnt.sc/ZQB7Md697VyX